esrally/reporter.py

```diff
@@ -79,7 +79,7 @@ def print_header(message):
     print_internal(console.format.bold(message))
 
 
-def write_single_report(report_file, report_format, cwd, headers, data_plain, data_rich, write_header=True):
+def write_single_report(report_file, report_format, cwd, headers, data_plain, data_rich):
```

Comment:
There is another callsite in `ComparisonReporter` that still passes this parameter.